### DISCUSSION TITLE
Tables

### DISCUSSION THREAD

**Discussion by CaiB** [thuliteio/doks#63] [Q&A] [answered]
I'm working on moving some documentation for a project of mine into a website using Doks. So far things are going OK. The biggest question I have however, is how do I create a table? I'm used to the GitHub Markdown syntax of `|---|` etc, but that doesn't appear to work.

Thanks!

**Comment by CaiB** [comment]
I ended up manually making the table with `<table>`, `<tr>`, etc. Please let me know if there is a better method.

**Comment by h-enk** [comment, marked as answer]
Hello @CaiB,

Tables in markdown are converted [per default](https://gohugo.io/getting-started/configuration-markup/#goldmark). So, I suppose you mean the [styling](https://getbootstrap.com/docs/4.5/content/tables/).

You could add e.g. the following to e.g. `./assets/scss/common/_global.scss`:

```
table {
  @extend .table;
  margin: 3rem 0;
}
```

Then...

![2021-01-02_13-16-54](https://user-images.githubusercontent.com/3902872/103457147-dc836400-4cfc-11eb-8935-0089c338020d.png)

...will result in:

![2021-01-02_13-17-59](https://user-images.githubusercontent.com/3902872/103457162-fcb32300-4cfc-11eb-9687-53c19e6f9c86.png)

**Reply by CaiB** [reply]
Gotcha, thank you. I must have simply made a mistake in my markdown syntax.

**Reply by epogrebnyak** [reply]
Markdown tables are notoriously hard to write and maintain, so a solution with html is perfectly workable.

**Comment by ccthecode** [comment]
Hi,

 you can visit https://www.tablesgenerator.com/markdown_tables

 ,type in your field manually like you would in Excel sheet and it will generate the markdown version of it.

Simply copy and paste.

The HTML tab above is also handy for generating HTML tables easily.
